closure/goog/testing/propertyreplacer.js

```diff
@@ -116,6 +116,7 @@ goog.testing.PropertyReplacer.hasKey_ = function(obj, key) {
     //   var dummy = document.body.constructor.prototype.className
     //   [Exception... "Cannot modify properties of a WrappedNative"]
     // </pre>
+    /** @suppress {unusedLocalVariables} */
```

Comment:
I removed that suppression for now. In fact most of the lint warnings in Closure Library are "@suppress is not allowed here". I think we need to allow more suppressions in the Compiler: https://github.com/google/closure-compiler/issues/1615.